components/tab-panel/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { partial, noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { default as withInstanceId } from '../higher-order/with-instance-id';
```

Comment:
You don't need `{ default as ... }` syntax here. This can be simplified to:

```js
import withInstanceId from '../higher-order/with-instance-id';
```

The default export is inferred when assigning outside the curly braces.